#keyframe.msg
geometry_msgs/PoseArray keyframePoses
std_msgs/Float64MultiArray[] covariances